index.html

```diff
@@ -149,6 +146,54 @@ <h2>
   ]
 }
         </pre>
+        
+        <pre class="example" title="Example did:web DID document using JWS2020">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
+  "verificationMethod": [
+    {
+      "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn#z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```

Comment:
```suggestion
      "id": "did:web:example.gov#z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```

Reply:
this has to match the did.... so maybe don't take my suggestion.

Reply:
yeah - this is such a common use case wasn't sure how we wanted to handle

Reply:
this is ok, you can use a less confusing fragment if you wish.